src/coreclr/tests/scripts/run-corefx-tests.bat

```diff
@@ -27,7 +27,7 @@ echo control, but is easy to implement. This file should be a list of assembly n
 echo without filename extension, one per line, e.g.:
 echo.
 echo     System.Console.Tests
-echo     System.Data.SqlClient.Tests
+echo     System.Data.Common.Tests
```

Comment:
I can't comment on it, but there's a `src/libraries/Native/pkg/_.pdb` in the diff file list, showing as "No changes".  Do you know what's going on there?

Reply:
The whole src/libraries/Native/pkg directory is gone. `_pdb` was an empty file. I guess it is what makes gihub to show the file this way.